functions.php

```diff
@@ -442,9 +442,14 @@ function odm_return_all_topics( $query ) {
 
 function odm_search_pre_get_posts($query)
 {
+  if(get_post_type()){
+    $post_type = $query->query['post_type'][0];
```

Comment:
Well, these lines of code is used to request wp to query posts be specify post type.
If we don't add these, it will query posts from all type of posts even the post_type is set (
eg. 'post_type' => 'map-layer') 
Sometimes, the news posts were appeared in map-explorer page.